CSS/main.css

```diff
@@ -1,2 +1,72 @@
-.footer{color:pink}
+body {
+  font-family: 'Raleway', sans-serif; }
+
+.section--headline {
+  text-align: center;
+  margin-top: 40px;
+  margin-bottom: 20px; }
+  .section--headline__title {
+    font-size: 34px;
+    text-transform: capitalize; }
+  .section--headline__subtitle {
+    font-size: 18px;
+    color: #4c545f; }
+    .section--headline__subtitle:before {
+      width: 75px;
+      height: 1px;
+      margin-right: 20px;
+      background: #4c545f;
+      content: '';
+      display: inline-block;
+      top: -4px;
+      position: relative; }
+    .section--headline__subtitle:after {
+      width: 75px;
+      height: 1px;
+      margin-left: 20px;
+      background: #4c545f;
+      content: '';
+      display: inline-block;
+      top: -4px;
+      position: relative; }
+
+.advantages--box {
+  font-family: 'Raleway', sans-serif;
```

Comment:
czy na pewno potrzebujemy dodawać to font-family? skoro mamy dodany już do znacznika **body**